.trivyignore.yaml

```diff
@@ -39,6 +39,7 @@ vulnerabilities:
 
 misconfigurations:
 - id: AVD-GIT-0001
+- id: AVD-GIT-0004
```

Comment:
are you able to provide a comment here  (wtih # ) why we are suppressing it?

Reply:
Yes I will add that, not least because we will want to remove this rule when MPE has require_signed_commits enabled in the future.